_posts/data-announces-formation.md

```diff
@@ -1,23 +1,22 @@
 ---
-title: "Formation of the Dallas Area Transit Alliance (DATA)"
-excerpt: "[DALLAS, Texas, July 30, 2024 —] Today we officially announce the formation of the Dallas Area Transit Alliance (DATA), a grassroots organization committed to the preservation and advancement of Dallas Area Rapid Transit (DART)."
-coverImage: "assets/blog/data-announces-formation/logo-transparent.png"
+title: "Formation of Dallas Area Transit Alliance (DATA)"
+excerpt: "Today we officially announce the formation of the Dallas Area Transit Alliance (DATA), a grassroots organization committed to the preservation and advancement of Dallas Area Rapid Transit (DART)."
+coverImage: "/assets/blog/data-announces-formation/data-meeting-july-20.webp"
 date: "2024-07-30T05:00:00.000Z"
 author:
-  name: DATA Team
-  picture: "/assets/blog/authors/jj.jpeg"
+  name: DATA Press Release
+  picture: "/assets/blog/authors/data-team.png"
 ogImage:
-  url: "assets/blog/data-announces-formation/logo-transparent.png"
+  url: "/assets/blog/data-announces-formation/data-meeting-july-20.webp"
+banner: "[Read our latest press release announcing our formation.]($slug)"
```

Comment:
The format of this is still changing so it will need to be updated when I'm done with my feature. @LEQUI000

Reply:
Got it. Will be on the lookout.